src/imp/libc/io/types.rs

```diff
@@ -259,6 +259,23 @@ bitflags! {
     }
 }
 
+#[cfg(not(target_os = "wasi"))]
+bitflags! {
+    /// `MS_*` flags for use with [`msync`].
+    ///
+    /// [`msync`]: crate::io::msync
+    pub struct MsyncFlags: i32 {
+        /// `MS_SYNC` Requests an update and waits for it to complete.
```

Comment:
Looking at the other `bitflags!` in this crate, it seems like in some cases the doc comment just repeats the identifier - probably to avoid all the additional work of copying documentation in.

But rustdoc doesn't require us to repeat the identifier name.  I think it'd be cleaner to drop it.  Like this:

```suggestion
        /// Requests an update and waits for it to complete.
```

And similarly elsewhere.

Reply:
The idea behind including the C identifier is to make it easier for people porting libc or C code, people familiar with the C APIs, and people looking at the POSIX/Linux/etc. docs which rustix links to, to help them search for things and see how things correspond.

In this specific case, "Requests an update and waits for it to complete" is a fine summary comment, but if someone really needs to know the fine details, they'll need to consult other sources. It'd indeed be cleaner if rustix could have self-sufficient documentation, but, as you say, that's a lot of additional work. We may do that work some day, but for now, I think it makes sense to include the C identifiers.

Reply:
OK, makes sense.  Though what may be even better is adding it as [an alias](https://doc.rust-lang.org/rustdoc/advanced-features.html#add-aliases-for-an-item-in-documentation-search)?